src/components/Admin/Import.vue

```diff
@@ -168,8 +168,14 @@ export default {
     },
 
     loadFile (e) {
-      var file = e.target.files[0]
-      if (file) {
+      let file = {}
+      if (e.type === 'drop') {
+        file = e.dataTransfer
+      } else {
+        file = e.target
+      }
+
+      if (file.files[0]) {
```

Comment:
Hopefully files is never unndefined, otherwise this will raise an error.

You could do `const { files = [] } = (e.type === 'drop' ? e.dataTransfer : e.target) || {}`
This way its always defined